nservicebus/upgrades/supported-platforms_content_core_7.partial.md

```diff
@@ -0,0 +1,10 @@
+| Framework | Version | Platform | Support |
+|------------------|:-------:|:--------:|:-------:|
+| .NET Framework | 4.5.2 or later<sup>1</sup> | [Windows](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies) | Supported |
+| .NET Core | 3.1 (LTS) | [Windows / Linux](https://github.com/dotnet/core/blob/master/release-notes/3.1/3.1-supported-os.md) | [Supported until Dec. 3, 2022](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) |
+| .NET | 5.0 | [Windows / Linux](https://github.com/dotnet/core/blob/main/release-notes/5.0/5.0-supported-os.md) | [Supported until May 8, 2022](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) |
+| .NET | 6.0 (LTS) | [Windows / Linux](https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md) | [Supported until Nov. 8, 2024](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)  |
+
+<sup>1</sup> Some packages may require a later version of the .NET Framework.
+
+macOS is supported for development purposes but not for production workloads.
```

Comment:
We could make this `<sup>2</sup>` and mark it for the Windows / Linux entries the 3 where it is true.

Reply:
Don't like it. Can't get it to look good with the little icon that's already superscripted to denote an off-site link.

Reply:
ok, was worth a try at least.